src/utils/constantsV3.ts

```diff
@@ -1,26 +1,10 @@
 import { Address } from 'viem';
 import { ChainId } from './constants';
 
-// V3
-export const STABLE_POOL_FACTORY_BALANCER_V3: Record<number, Address> = {
-    [ChainId.SEPOLIA]: '0xd67F485C07D258B3e93835a3799d862ffcB55923',
-    [ChainId.MAINNET]: '0xB9d01CA61b9C181dA1051bFDd28e1097e920AB14',
-    [ChainId.GNOSIS_CHAIN]: '0x22625eEDd92c81a219A83e1dc48f88d54786B017',
-    [ChainId.SONIC]: '0x815Ab57a5a2E4976cEC0b43C2D50CF26EF6F31Fd',
-    [ChainId.ARBITRUM_ONE]: '0xEB1eeaBF0126d813589C3D2CfeFFE410D9aE3863',
-    [ChainId.BASE]: '0xb9F8AB3ED3F3aCBa64Bc6cd2DcA74B7F38fD7B88',
-    [ChainId.MONAD_TESTNET]: '0xb9F8AB3ED3F3aCBa64Bc6cd2DcA74B7F38fD7B88',
-    [ChainId.AVALANCHE]: '0x1702067424096F07A60e62cceE3dE9420068492D',
-};
-
-// V3
-export const STABLE_SURGE_FACTORY: Record<number, Address> = {
-    [ChainId.ARBITRUM_ONE]: '0x86e67E115f96DF37239E0479441303De0de7bc2b',
-    [ChainId.BASE]: '0x4fb47126Fa83A8734991E41B942Ac29A3266C968',
-    [ChainId.GNOSIS_CHAIN]: '0x268E2EE1413D768b6e2dc3F5a4ddc9Ae03d9AF42',
-    [ChainId.MAINNET]: '0xD53F5d8d926fb2a0f7Be614B16e649B8aC102D83',
-    [ChainId.SEPOLIA]: '0xD516c344413B4282dF1E4082EAE6B1081F3b1932',
-    // TODO once deployed [ChainId.AVALANCHE]: '0x0..',
+export const STABLE_POOL_CONSTRAINTS = {
+    MAX_TOKENS: 5,
+    MIN_AMP: 1n,
+    MAX_AMP: 50000n,
```

Comment:
Max amp moved from `5000n` to `50000n`

https://github.com/balancer/balancer-v3-monorepo/blob/6bf35d1d6be50551a1dd3d355dfd384d8501101c/pkg/solidity-utils/contracts/math/StableMath.sol#L34